src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -164,6 +164,9 @@ export default class DefaultContentShareController
           return;
         }
         const stream = this.mediaStreamBroker.mediaStream;
+        this.contentAudioVideo.logger.info(
```

Comment:
This should be just this.logger.info

Reply:
DefaultContentShareController.ts does not have direct reference to the logger.